src/main/scala/it/pagopa/interop/partyprocess/api/impl/ProcessApiServiceImpl.scala

```diff
@@ -367,7 +377,48 @@ class ProcessApiServiceImpl(
       ApplicationConfiguration.onboardingMailTaxCodePlaceholder     -> currentUser.externalId
     )
 
-    val bodyParameters: Map[String, String] = tokenParameters ++ userParameters
+    val institutionInfoParameters: scala.collection.mutable.Map[String, String] = scala.collection.mutable.Map()
+    if (onboardingRequest.institutionUpdate != null) {
```

Comment:
you don't need this check, `map` executes the given function only if the field is not `None`.

(btw in Scala an empty `Option` is `None`, not `null`)